chapter_convolutional-neural-networks/conv-layer.md

```diff
@@ -175,10 +194,15 @@ Y
 不出所料，这个[**卷积核`K`只可以检测垂直边缘**]，无法检测水平边缘。
 
 ```{.python .input}
-#@tab all
+#@tab mxnet, pytorch, tensorflow
 corr2d(d2l.transpose(X), K)
 ```
 
+```{.python .input}
```

Comment:
Reuse code here. We may need to revise config.ini to specify the alias.

Reply:
resolved